app/components/SelectWallet.tsx

```diff
@@ -11,7 +12,12 @@ export default function SelectWallet() {
         <button
           disabled={!connector.ready}
           key={connector.id}
-          onClick={() => connect(connector)}
+          onClick={() => {
+            console.log('connected', isLoading)
```

Comment:
probably could remove these logs
I'm curious why connector:connector works but not connector, hmmm or is this another instance of debugging which would make sense